### PR TITLE
Missing migration file

### PR DESCRIPTION
**Pull request by bartoszpietrzak1994**

### CODE REVIEW

File: src/Migrations/Version20190515085648.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
```
Comment (Zales0123) [inline src/Migrations/Version20190515085648.php]: I would remove this redundant comment

File: src/Migrations/Version20190515085648.php [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+final class Version20190515085648 extends AbstractMigration
+{
+    public function getDescription() : string
+    {
+        return '';
+    }
```
Comment (Zales0123) [inline src/Migrations/Version20190515085648.php]: And this redundant method

File: src/Migrations/Version20190515085648.php [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20190515085648 extends AbstractMigration
+{
+    public function up(Schema $schema) : void
```
Comment (GSadee) [inline src/Migrations/Version20190515085648.php]: Unnecessary space after `(`

File: src/Migrations/Version20190515085648.php [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        $this->addSql('ALTER TABLE sylius_channel DROP FOREIGN KEY FK_16C8119E5CDB2AEB');
+        $this->addSql('DROP TABLE sylius_invoicing_plugin_shop_billing_data');
+        $this->addSql('ALTER TABLE sylius_invoicing_plugin_invoice CHANGE channel_code channel_code VARCHAR(255) NOT NULL, CHANGE channel_name channel_name VARCHAR(255) NOT NULL');
+        $this->addSql('CREATE INDEX IDX_6196A1F9A393D2FB43625D9F ON sylius_order (state, updated_at)');
+        $this->addSql('ALTER TABLE sylius_promotion_coupon ADD reusable_from_cancelled_orders TINYINT(1) DEFAULT \'1\' NOT NULL');
+        $this->addSql('DROP INDEX UNIQ_16C8119E5CDB2AEB ON sylius_channel');
+        $this->addSql('ALTER TABLE sylius_channel DROP billing_data_id, DROP taxId');
+    }
+
+    public function down(Schema $schema) : void
```
Comment (GSadee) [inline src/Migrations/Version20190515085648.php]: As above

**Pull request merged**